3d Space Shooter/Assets/Scripts/Ship_Selector_Toggle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Ship_Selector_Toggle : MonoBehaviour {
+
+	public GameObject shipSelector;
+	public Canvas healthBarDisplay;
+	public Image healthBar;
+
+	// Use this for initialization
+	void Start ()
+	{
+		shipSelector.SetActive (false);
+		healthBarDisplay.enabled = false;
+	}
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
Notice there is no function here for finding an enemy object, those have to be hand placed in by someone and it will only support a single instance of an enemy.